lib/metanorma/plugin/datastruct/yaml2_text_preprocessor.rb

```diff
@@ -39,8 +39,15 @@ def initialize(config = {})
 
         # https://ruby-doc.org/stdlib-2.5.1/libdoc/psych/rdoc/Psych.html#method-c-safe_load
         def content_from_file(document, file_path)
+          resolved_file_path = relative_file_path(document, file_path)
+
+          unless File.exist?(resolved_file_path)
+            document.logger.warn("YAML file not found: #{resolved_file_path}")
```

Comment:
@kwkwan can you please help the user also know what block this YAML was referenced? e.g.

```ruby
warn("YAML file referenced in [yaml2text] block not found: xxx")
```